packages/devops/scripts/ci/testInternalDependencies.sh

```diff
@@ -1,13 +1,11 @@
-#!/bin/bash
+#!/bin/bash -le
 DIR=$(dirname "$0")
-${DIR}/copyFromCommonVolume.sh
+# ${DIR}/copyFromCommonVolume.sh
 for PACKAGE in $(${DIR}/../../../../scripts/bash/getInternalDependencies.sh); do
     # skip the following packages - they get tested separately as they require db access
     if [[ "$PACKAGE" == "database" || "$PACKAGE" == "data-api" || "$PACKAGE" == "auth" || "$PACKAGE" == "indicators" || "$PACKAGE" == "server-boilerplate" ]]; then
         continue
     fi
     echo Testing ${PACKAGE}
-    if ! yarn workspace @tupaia/${PACKAGE} test; then
-        exit 1 # the tests for this internal depencency failed
-    fi
+    yarn workspace @tupaia/${PACKAGE} test
```

Comment:
No longer need the conditional given that the `-e` means it'll exit the script early anyway